modules/vcf/templates/annotate.sh

```diff
@@ -180,9 +234,12 @@ main () {
     annot_sv
   fi
 
+  if [ -n "!{hpoIds}" ]; then
```

Comment:
line 206 and this line should use the same check?

Reply:
than you will never get the warnings